src/forms/__snapshots__/toggleSwitch.test.jsx.snap

```diff
@@ -1,32 +1,13 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ToggleSwitch basic renders checked toggle switch 1`] = `
-<span
-  aria-pressed={false}
-  onKeyUp={[Function]}
-  role="button"
-  tabIndex="0"
->
-  <ToggleSwitch
-    checked={true}
-    isActive={false}
-    name="testToggle"
-    toggleActive={[Function]}
-  />
-</span>
+<ToggleWrapper
```

Comment:
We lost some coverage here - could using `mount` instead of `shallow` help?

Reply:
Using `mount` instead of `shallow` didn't bring coverage up

Reply:
Let's to a follow up ticket to see if we can do anything to improve the snapshot. This otherwise looks good to go